blazor/heatmap-chart/selection.md

```diff
@@ -9,7 +9,15 @@ documentation: ug
 
 # Selection in Blazor HeatMap Chart Component
 
-In the [Blazor HeatMap Chart](https://www.syncfusion.com/blazor-components/blazor-heatmap-chart), the cell selection is used to select the single or multiple heat map cells at runtime and get the selected cell details using the `CellSelected`  event. You can enable this cell selection using the `AllowSelection` property.
+In the [Blazor HeatMap Chart](https://www.syncfusion.com/blazor-components/blazor-heatmap-chart), the cell selection is used to select the single or multiple HeatMap cells at runtime and get the selected cell details using the `CellSelected`  event. You can enable this cell selection using the `AllowSelection` property.
+
+To select multiple cells using different modes of interactions as shown in the below table.
+
+|   Modes of Interactions |   Description                                                                                                      |
+|------------------------ | -------------------------------------------------------------------------------------------------------------------|
+|   Mouse                 |  To select multiple cells, click and drag the mouse across the specific cells.                                     |
+|   Touch                 |  To select multiple cells, touch and drag the fingers across the specific cells.                                   |
+|   Keyboard              |  To select multiple cells, holding the Ctrl (Control) key on your keyboard and click on any other specific cells.  |
```

Comment:
The Ctrl key on the keyboard can be used to enable multiple cell selection with mouse and touch interaction.